lrmd/lrmd.c

```diff
@@ -576,12 +576,12 @@ lsb2uniform_rc(const char *action, int rc)
             return PCMK_OCF_OK;
         case PCMK_LSB_STATUS_NOT_INSTALLED:
             return PCMK_OCF_NOT_INSTALLED;
+        case PCMK_LSB_STATUS_INSUFFICIENT_PRIV:
+            return PCMK_OCF_INSUFFICIENT_PRIV;
         case PCMK_LSB_STATUS_VAR_PID:
         case PCMK_LSB_STATUS_VAR_LOCK:
         case PCMK_LSB_STATUS_NOT_RUNNING:
             return PCMK_OCF_NOT_RUNNING;
-        default:
```

Comment:
The behavior is the same; without default, it falls through to the next line which is return PCMK_OCF_UNKNOWN_ERROR. I can add it back if you think it's more readable, but in that case the final return is unreachable.